integration-test/src/main/java/org/apache/iotdb/it/env/AbstractEnv.java

```diff
@@ -185,12 +187,40 @@ public void testWorking() {
     try {
       long startTime = System.currentTimeMillis();
       testDelegate.requestAll();
+      if (!configNodeWrapperList.isEmpty()) {
+        checkNodeHeartbeat();
+      }
       logger.info("Start cluster costs: {}s", (System.currentTimeMillis() - startTime) / 1000.0);
     } catch (Exception e) {
       fail("After 30 times retry, the cluster can't work!");
     }
   }
 
+  private void checkNodeHeartbeat() {
+    TShowClusterResp showClusterResp;
+    try (SyncConfigNodeIServiceClient client =
+        (SyncConfigNodeIServiceClient) getConfigNodeConnection()) {
+      boolean flag;
+      for (int i = 0; i < 30; i++) {
+        flag = true;
+        showClusterResp = client.showCluster();
+        Map<Integer, String> nodeStatus = showClusterResp.getNodeStatus();
+        for (String status : nodeStatus.values()) {
+          if (!status.equals("Running")) {
+            flag = false;
+            break;
+          }
+        }
+        if (flag) {
+          return;
+        }
+        Thread.sleep(1000);
+      }
+    } catch (Exception e) {
+      fail("Not all nodes are in the Running states");
```

Comment:
If we need sleeping some time to wait that all the status of nodes have been running?